### PR TITLE
privatize the openshift-tests-private repo logs

### PR DESCRIPTION
**Pull request by jianzhangbjz**
@smg247 Many thanks for your support! One question, how can we add multi private repos there? Thanks! Such as, another one: https://github.com/openshift/cucushift

### CODE REVIEW

**Comment by smg247** [comment]
> @smg247 Many thanks for your support! One question, how can we add multi private repos there? Thanks! Such as, another one: https://github.com/openshift/cucushift

You should be able to handle that by just adding another entry to the `prowjob_default_entries` list like:
```
- cluster: '*'
  config:
    tenant_id: qe-private
  repo: openshift/openshift-tests-private
```

Note: please don't remove my `smg247/ci-docs` repo as I need that for additional testing. There are a couple of minor outstanding bugs that I am still working on here.

**Comment by jianzhangbjz** [comment]
@smg247 Many thanks for your review and comments, updated it.

**Review by smg247 — APPROVED** [review]

File: core-services/sync-rover-groups/_config.yaml
Comment (jianzhangbjz) [inline core-services/sync-rover-groups/_config.yaml:17]: @smg247 For this [openshift/openshift-tests-private](https://github.com/openshift/openshift-tests-private) Prow test, seems like it uses the `build03` cluster, such as https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_openshift-tests-private/2689/pull-ci-openshift-openshift-tests-private-master-e2e-aws/1493230897246244864
So, I add all `build0*` cluster here. Could you help have a review when you get a chance? Thanks!

File: ci-operator/jobs/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.7-presubmits.yaml
Comment (jianzhangbjz) [inline ci-operator/jobs/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.7-presubmits.yaml:181]: @smg247 After running `make update`, here has been changed to `packet-edge` from `packet`, not sure why.

Comment (jianzhangbjz) [reply ci-operator/jobs/openshift-priv/cluster-baremetal-operator/openshift-priv-cluster-baremetal-operator-release-4.7-presubmits.yaml]: Never mind, I addressed it by rebasing master branch.

File: core-services/sync-rover-groups/_config.yaml
Comment (jianzhangbjz) [reply core-services/sync-rover-groups/_config.yaml:17]: Once this PR merge, the cluster will point to `qe-priovate-deck`, such as  https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/, so no need the `build01-04` clusters here. cc: @kuiwang02

File: core-services/sync-rover-groups/_config.yaml
Comment (jianzhangbjz) [inline core-services/sync-rover-groups/_config.yaml:16]: @smg247 Since this private URL is https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/, I think here should be `apps.ci`, right?

Comment (jianzhangbjz) [reply core-services/sync-rover-groups/_config.yaml:16]: ok, thanks!

Comment (jianzhangbjz) [reply core-services/sync-rover-groups/_config.yaml]: @smg247 One more question, now the job [openshift-openshift-tests-private-master-presubmits.yaml](https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-master-presubmits.yaml#L8) uses the `build02` cluster, do I need to add `build01 - build04` here? Or this build farm cluster will be changed to `app.ci` automatically once this PR is merged? Thanks!

Comment (jianzhangbjz) [reply core-services/sync-rover-groups/_config.yaml]: If no need, could you help add `/lgtm` label? Thanks!

Comment (smg247) [reply core-services/sync-rover-groups/_config.yaml]: The job itself will run on the build02 cluster, but you will access the logs using the new qe-private-deck instance running on app.ci. There is no need to sync the group to any of the other clusters.

**Pull request merged**
